src/__tests__/DateTime.test.js

```diff
@@ -2,7 +2,7 @@
 
 import { Kind } from 'graphql/language';
 
-import { DateTime } from '../';
+import { DateTime } from '..';
```

Comment:
I had no idea that this was possible. 🤦‍♂️  😂 
Learnt something new today. 🥇

Reply:
Me too.